source/scripting/RelationshipGroup.cs

```diff
@@ -34,15 +34,15 @@ public ulong NativeValue
 			}
 		}
 
-		Relationship GetRelationshipBetweenGroups(RelationshipGroup targetGroup)
+		public Relationship GetRelationshipBetweenGroups(RelationshipGroup targetGroup)
 		{
 			return Function.Call<Relationship>(Native.Hash.GET_RELATIONSHIP_BETWEEN_GROUPS, Hash, targetGroup);
 		}
-		void SetRelationshipBetweenGroups(RelationshipGroup targetGroup, Relationship relationship)
+		public void SetRelationshipBetweenGroups(RelationshipGroup targetGroup, Relationship relationship)
 		{
 			SetRelationshipBetweenGroups(targetGroup, relationship);
```

Comment:
Stack overflow here, can only guess you're meant to be calling the function below